packages/driver/cypress/e2e/cypress/stack_utils.cy.js

```diff
@@ -93,6 +126,15 @@ describe('driver/src/cypress/stack_utils', () => {
 
       expect($stackUtils.getCodeFrame(originalErr)).to.be.undefined
     })
+
+    it('returns relativePath from root when `absoluteFile` starts with `repoRoot`', () => {
+      // question - how do i mock a value in the cypress config?
```

Comment:
Here I'd like to set the value of `Cypress.config('repoRoot')`. 

I've tried... 
1. Setting it directly via `Cypress.config('repoRoot, 'dev')`. This didn't work -- I think b/c Tim and I made this a non-configurable value, similar to `projectRoot`. 
2. Stubbing it via `cy.stub(Cypress, 'config').toFakeFn(() => ({ ...Cypress.config(), repoRoot: 'dev' }))`. This ended up crashing Cypress with an infinite loop of uncaught errors from lodash 👹

Reply:
I think this is actually getting set how'd you think:

```js
      Cypress.config('repoRoot', 'dev')
      console.log(Cypress.config('repoRoot')) // logs 'dev'
```

Reply:
Also I think this should be `Cypress.config('repoRoot, '/dev')`, right (note the leading `/`). Your test can't pass otherwise, since `absoluteFile` always starts wtih `/dev`.

Reply:
You figured it out!!!! I was missing the leading `/`. Thank you 😻